alts/src/generated/main/grpc/io/grpc/alts/internal/HandshakerServiceGrpc.java

```diff
@@ -92,7 +92,7 @@ public HandshakerServiceFutureStub newStub(io.grpc.Channel channel, io.grpc.Call
 
   /**
    */
-  public static abstract class HandshakerServiceImplBase implements io.grpc.BindableService {
+  public interface HandshakerServiceAsync {
```

Comment:
How does someone use this interface? They implement it, then what? It seems we need to expose a `bindService(HandshakerServiceAsync)` somewhere.

Reply:
Created a bindService(<service>Async) at the end of the class.